src/vs/platform/telemetry/node/1dsAppender.ts

```diff
@@ -0,0 +1,158 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import type { IExtendedConfiguration, AppInsightsCore } from '@microsoft/1ds-core-js';
+import { IChannelConfiguration, IPayloadData, IXHROverride, PostChannel } from '@microsoft/1ds-post-js';
+import * as https from 'https';
+import { onUnexpectedError } from 'vs/base/common/errors';
+import { mixin } from 'vs/base/common/objects';
+import { ITelemetryAppender, validateTelemetryData } from 'vs/platform/telemetry/common/telemetryUtils';
+
+async function getClient(instrumentationKey: string): Promise<AppInsightsCore> {
+	const oneDs = await import('@microsoft/1ds-core-js');
+	const appInsightsCore = new oneDs.AppInsightsCore();
+	const collectorChannelPlugin: PostChannel = new PostChannel();
+	// Configure the app insights core to send to collector++ and disable logging of debug info
+	const coreConfig: IExtendedConfiguration = {
+		instrumentationKey,
+		endpointUrl: 'https://mobile.events.data.microsoft.com/collect/v1',
```

Comment:
I assume that you have your own endpoint or you have checked with the collector team that this is a valid CNAME (as it's not a standard 1DS looking endpoint). And as 1DS now sends some additional information as headers vs Query string values (so that the OPTIONS call can be cached -- for browser scenarios) if the endpoint doesn't respond with the "allowed" set of headers the request will fail.

Reply:
I will check with Andrew Faust but this is what we were sending to before for collector++ and CS 2.0

Reply:
Generally, the 1DS collector endpoints are <domain>/OneCollector/1.0 as they have all of the additional Cors (OPTIONS) responses while the others don't.